pkgs/development/python-modules/loguru/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib
 , stdenv
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , fetchpatch
```

Comment:
```suggestion
```

Reply:
@evanjs Can you do this already?

Reply:
Yep, forgot to push my change from a few days ago. Oops.